CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 
  - Compilation error when there are duplicated element id
  - `ComboBox` now has a `selected` callback
+ - Added `sixtyfps::Weak::upgrade_in_event_loop' in the Rust API
```

Comment:
Maybe this or something more elaborate should also be in the commit message, for less editing in the this-week summary :)